automatic/curl/README.md

```diff
@@ -0,0 +1,7 @@
+# <img src=""></img> [curl](https://chocolatey.com/packages/curl)
```

Comment:
however, the Readme.md file wont get updated if the nuspec file is already updated with the latest icon url